### ISSUE TITLE
macOS tags are not preserved when synced with Maestral  

### ISSUE THREAD

**Issue by christophe999** [samschott/maestral#590]
**Describe the bug**
On MacOS with Maestro, tag a synced file.
The tag metadata is not synced.

**To Reproduce**
Tag a sync file.
Check on another computer, the tag is lost.

**Expected behaviour**
With the dropbox client, tags are preserved.

**System:**
Installed from the DMG file.
My main Python does not have the QT modules.

**Additional context**

**Comment by samschott** [comment]
Yeah, this is a known limitation. The public Dropbox API does not allow querying for or setting some filesystem-specific metadata such as tags. Their private API does allow this, therefore reserving such capabilities for their own client. I've flagged it as a feature request but its unlikely that they will change this soon, if at all.

**Comment by christophe999** [comment]
Thanks @samschott 
I understand you are limited by the public API.
Except for this particular requirement of mine, Maestro is the perfect client IMHO. 
Congrat and thanks to the contributors.
